wp/lib/bap_wp/src/precondition.ml

```diff
@@ -1397,47 +1401,36 @@ let in_loop (loop : Graphs.Ir.Node.t group) (tid : Tid.t) : bool =
    - for each node in the SCC, find its parent in the dominator tree
    - if the parent node is not in the original SCC, it is an exit node.
 
-   The exit node of a loop is the first node outside of the SCC in the control
-   flow graph. This should be the node the loop header points to that isn't a
-   part of the loop.
+   The exit node of a loop is a node outside of the SCC in the control flow
+   graph. This node will always be visited when exiting a loop.
```

Comment:
Not sure what that last sentence means. Is it "will have already been visited"?

Reply:
The execution of the program will always go through the exit node if we are on the path that contains the loop.

Reply:
Is this true though? What if a loop has 2 exits?

Reply:
Hmm.. I guess not always true. I only had a few samples with multiple exits, and this always returned the node that all the exits pointed to. I can't say for other loop shapes.

Reply:
So we don't actually know what this code does?